src/components/editor/property/OutlineHeader.jsx

```diff
@@ -40,10 +42,41 @@ OutlineHeader.propTypes = {
   collapsed: PropTypes.any,
   handleCollapsed: PropTypes.func,
   id: PropTypes.string,
-  isRequired: PropTypes.any,
-  isAdd: PropTypes.bool,
-  pt: PropTypes.object,
-  spacer: PropTypes.oneOfType([PropTypes.string, PropTypes.number]),
+  property: PropTypes.object,
+  reduxPath: PropTypes.array,
+  resourceModel: PropTypes.object,
+  handleTogglePlusButton: PropTypes.func,
+  handleRemoveButton: PropTypes.func,
 }
 
-export default OutlineHeader
+const mapStateToProps = (state, ourProps) => {
+  const propertyURI = ourProps.reduxPath.slice(-1)[0]
+  const resourceTemplateId = ourProps.reduxPath.slice(-2)[0]
+  const property = getPropertyTemplate(state, resourceTemplateId, propertyURI)
+  const resourceModel = findNode(state.selectorReducer, ourProps.reduxPath)
+
+  return {
+    resourceModel,
+    property,
+    collapsed: !isExpanded(state.selectorReducer, ourProps.reduxPath),
+  }
+}
+
+const mapDispatchToProps = (dispatch, ownProps) => ({
+  handleTogglePlusButton: (event) => {
```

Comment:
As this is not a "plus" button anymore, can you please change the name to "handleToggleCollapse" or something without "plus" in it.